[2023-01-05T13:28:40.443+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.add_datalake_quality_ctrl_step manual__2023-01-05T10:56:03.364986+00:00 [queued]>
[2023-01-05T13:28:40.460+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.add_datalake_quality_ctrl_step manual__2023-01-05T10:56:03.364986+00:00 [queued]>
[2023-01-05T13:28:40.460+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-01-05T13:28:40.460+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 3
[2023-01-05T13:28:40.460+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-01-05T13:28:40.474+0000] {taskinstance.py:1389} INFO - Executing <Task(EmrAddStepsOperator): add_datalake_quality_ctrl_step> on 2023-01-05 10:56:03.364986+00:00
[2023-01-05T13:28:40.478+0000] {standard_task_runner.py:52} INFO - Started process 402 to run task
[2023-01-05T13:28:40.482+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'atn_update_datalake', 'add_datalake_quality_ctrl_step', 'manual__2023-01-05T10:56:03.364986+00:00', '--job-id', '3803', '--raw', '--subdir', 'DAGS_FOLDER/market_data_analytics/my_dags/datalake_update.py', '--cfg-path', '/tmp/tmpdky1ztvb', '--error-file', '/tmp/tmp5vj7vwnd']
[2023-01-05T13:28:40.482+0000] {standard_task_runner.py:80} INFO - Job 3803: Subtask add_datalake_quality_ctrl_step
[2023-01-05T13:28:40.550+0000] {task_command.py:371} INFO - Running <TaskInstance: atn_update_datalake.add_datalake_quality_ctrl_step manual__2023-01-05T10:56:03.364986+00:00 [running]> on host 370ed6868bc3
[2023-01-05T13:28:40.640+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=atn
AIRFLOW_CTX_DAG_ID=atn_update_datalake
AIRFLOW_CTX_TASK_ID=add_datalake_quality_ctrl_step
AIRFLOW_CTX_EXECUTION_DATE=2023-01-05T10:56:03.364986+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-05T10:56:03.364986+00:00
[2023-01-05T13:28:40.651+0000] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2023-01-05T13:28:40.652+0000] {connection_wrapper.py:210} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2023-01-05T13:28:40.722+0000] {emr.py:104} INFO - Adding steps to j-1Z5DDUK8PTRNA
[2023-01-05T13:28:48.579+0000] {emr.py:117} INFO - Steps ['s-MMZYKC9GIRTL'] added to JobFlow
[2023-01-05T13:28:48.631+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=atn_update_datalake, task_id=add_datalake_quality_ctrl_step, execution_date=20230105T105603, start_date=20230105T132840, end_date=20230105T132848
[2023-01-05T13:28:48.653+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-05T13:28:48.692+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
